pygeo/constraints/DVCon.py

```diff
@@ -1464,6 +1465,7 @@ def addTriangulatedSurfaceConstraint(
 
```

Comment:
I think `_getSurfaceVertices` function call may cause some issues if the strings are the same because it calls the surface based on the name provided. Can the default names be different in the `mphys_dvgeo` ?

Reply:
The surfaces don't have the same name, one has `surface_1_name=None` and the other has `surface_2_name="default"`.

Reply:
right, I mean in the mphys wrapper you define the `surface_1_name` and `surface_2_name` to be `default`. If the user doesn't change these names, will it be an issue? in your test function these surface names are different, but have you tested both surfaces' names to be `default`?

Reply:
Agreed, looks good to me!